### ISSUE TITLE
What about a text file starts with 'MZ'?

### ISSUE THREAD

**Issue by anders-liu** [kevva/is-exe#2]
I think you'd better add more check on the buffer, includes but not limited to:
- The buffer length should be at least 0x80 bytes, which is the minimum DOS stub of a modern PE file.
- Use DataView.getUint32(0x3C) to get a point say P, and use DataView.getUint32(P) to get the PE signature, and check whether it is 0x50450000.

**Comment by sindresorhus** [comment]
Agreed. Would be happy to accept a PR over at https://github.com/sindresorhus/file-type/blob/c4025ba2f47c188bdbacbc761254d3f11d2ff049/index.js#L294-L299, and we can mirror it over here afterwards.
